### PR TITLE
Bringing Facbebook\PseudiRandomString namespace into PSR2 compliance

### PR DESCRIPTION
**Pull request by stevenmaguire**
It's happening.

### CODE REVIEW

File: src/Facebook/PseudoRandomString/McryptPseudoRandomStringGenerator.php [hunk 1]
```diff
@@ -27,44 +27,42 @@
...
-      );
+        if ($binaryString === false) {
+            throw new FacebookSDKException(
+                static::ERROR_MESSAGE .
+                'mcrypt_create_iv() returned an error.'
+            );
+        }
+        return $this->binToHex($binaryString, $length);
     }
-
```
Comment (yguedidi) [inline src/Facebook/PseudoRandomString/McryptPseudoRandomStringGenerator.php]: You should keep this empty line

File: src/Facebook/PseudoRandomString/McryptPseudoRandomStringGenerator.php [hunk 2]
```diff
@@ -27,44 +27,42 @@
...
 class McryptPseudoRandomStringGenerator implements PseudoRandomStringGeneratorInterface
 {
+    use PseudoRandomStringGeneratorTrait;
 
-  use PseudoRandomStringGeneratorTrait;
+    /**
+     * @const string The error message when generating the string fails.
+     */
+    const ERROR_MESSAGE = 'Unable to generate a cryptographically secure '
+        .'pseudo-random string from mcrypt_create_iv(). ';
```
Comment (yguedidi) [inline src/Facebook/PseudoRandomString/McryptPseudoRandomStringGenerator.php]: One line IMO

File: src/Facebook/PseudoRandomString/McryptPseudoRandomStringGenerator.php (hunk 1 above)
Comment (stevenmaguire) [inline src/Facebook/PseudoRandomString/McryptPseudoRandomStringGenerator.php]: I can get behind that.

File: src/Facebook/PseudoRandomString/McryptPseudoRandomStringGenerator.php (hunk 2 above)
Comment (stevenmaguire) [inline src/Facebook/PseudoRandomString/McryptPseudoRandomStringGenerator.php]: Seeing as how there is no *hard* limit on lines, it is just a single string, and I enjoy playing nice with others.

**Pull request merged**